packages/govuk-frontend-review/src/views/layouts/_generic.njk

```diff
@@ -4,6 +4,10 @@
   <link rel="stylesheet" href="/stylesheets/app.min.css">
 
   {% block styles %}{% endblock %}
+
+  <script type="importmap">
+    { "imports": { "govuk-frontend": "/javascripts/all.bundle.min.mjs" } }
```

Comment:
An import map has been added to resolve `govuk-frontend` to `/javascripts/all.bundle.min.mjs` when run inside Puppeteer `page.evaluate()`

Reply:
**question** Is that importmap used outside of the test boilerplate? If not, I'd suggest moving it to that template specifically (even if it lives at the top of the `<body>` tag) as it's a feature to help with the tests 😊

Reply:
It's used by `renderAndInitialise()` via [/tests/boilerplate](http://govuk-frontend-review.herokuapp.com/tests/boilerplate)

https://github.com/alphagov/govuk-frontend/blob/bf728fc3876fe942e61356815323d37c9770717b/shared/helpers/puppeteer.js#L101

(Few other tests also use boilerplate)

But all the globals tests (now "exports" rather than globals) use the [home page `/`](http://govuk-frontend-review.herokuapp.com/)

https://github.com/alphagov/govuk-frontend/blob/bf728fc3876fe942e61356815323d37c9770717b/packages/govuk-frontend/src/govuk/components/globals.test.mjs#L52

https://github.com/alphagov/govuk-frontend/blob/bf728fc3876fe942e61356815323d37c9770717b/packages/govuk-frontend/src/govuk/components/globals.test.mjs#L20

https://github.com/alphagov/govuk-frontend/blob/bf728fc3876fe942e61356815323d37c9770717b/packages/govuk-frontend/src/govuk/components/globals.test.mjs#L12

Let me just double check they'd all work from the boilerplate too

Reply:
@romaricpascal Yeah all fine, I've moved it to the boilerplate only